docs/proposals/multiple-volumegroups.md

```diff
@@ -59,53 +62,106 @@ metadata:
 provisioner: topolvm.io
 parameters:
   "csi.storage.k8s.io/fstype": "xfs"
-  "topolvm.io/volume-group": "ssd"
+  "topolvm.io/device-class": "ssd"
+volumeBindingMode: WaitForFirstConsumer
+```
+
+`lvmd` receives the device class as a parameter.
+`lvmd` has a setting for each device class, can decide a target volume group for a given device class.
+`lvmd` will operate a logical volume on the target volume group.
```

Comment:
```suggestion
The device class name is then passed to `lvmd`.

`lvmd` has a mapping between device classes and LVM volume groups.
It can, therefore, create a logical volume in multiple volume groups.
```